lib/gui/models/supported-formats.js

```diff
@@ -128,25 +128,43 @@ SupportedFormats.service('SupportedFormatsModel', function() {
       return false;
     }
 
-    return this.isSupportedImage(path.basename(imagePath, `.${extension}`));
+    return this.isSupportedImage({
+      path: path.basename(image.path, `.${extension}`)
+    });
   };
 
   /**
    * @summary Check if an image seems to be a Windows image
    * @function
    * @public
    *
-   * @param {String} imagePath - image path
+   * @param {Object} image - image metadata
    * @returns {Boolean} whether the image seems to be a Windows image
    *
    * @example
-   * if (SupportedFormatsModel.looksLikeWindowsImage('path/to/en_windows_7_ultimate_with_sp1_x86_dvd_u_677460.iso')) {
+   * if (SupportedFormatsModel.looksLikeWindowsImage(image)) {
    *   console.log('Looks like a Windows image');
    * }
    */
-  this.looksLikeWindowsImage = (imagePath) => {
+  this.looksLikeWindowsImage = (image) => {
     const regex = /windows|win7|win8|win10|winxp/i;
-    return regex.test(path.basename(imagePath));
+    return regex.test(path.basename(image.path));
+  };
+
+  /**
+   * Check if an image is missing a partition table as an indicator for bootability
+   * @function
+   * @public
+   *
+   * @param {Object} image - image metadata
+   * @returns {Boolean} whether the image is missing a partition table
+   * @example
+   * if (SupportedFormatsModel.missingPartitionTable(image)) {
+   *   console.log('Looks like it might not boot');
+   * }
+   */
+  this.missingPartitionTable = (image) => {
+    return !image.hasMBR;
```

Comment:
For clarity, I wonder if it'd be better to do something 'more explicit' like https://github.com/resin-io/etcher/pull/1183/files#diff-dec8509d5c0e865bccdce35d2dc3c9ccR47 ?
````js
_.get(options, [ 'ignoreSleepUpdateCheck' ], false)
```